controllers/topology/deployment.go

```diff
@@ -303,10 +303,18 @@ func (r *DeploymentReconciler) renderDeploymentVolumes(
 			},
 		)
 
+		var mountPath string
+		// mount relative paths under /clabernetes, and absolute paths as is
+		if strings.HasPrefix(podVolume.FilePath, "/") {
+			mountPath = podVolume.FilePath
+		} else {
+			mountPath = fmt.Sprintf("/clabernetes/%s", podVolume.FilePath)
+		}
+
```

Comment:
just to be clear: the abs path stuff is a second thing in this pr right? like its all clabverter setup related not related to the global config thing ya? or is jetlag getting the better of me :D

Reply:
I packed two things in this PR

1. handling of abs paths (I will have to check if I did it only for licenses or for binds either...)
2. made an attempt to fix the way global config obj is initialized, because I noticed that setting the privLauncher to true has no effect on the global config

Reply:
ok, cool, then I am tracking :D ty ty